docs/02-certificate-authority.md

```diff
@@ -137,6 +137,12 @@ KUBERNETES_PUBLIC_ADDRESS=$(aws elb describe-load-balancers \
   jq -r '.LoadBalancerDescriptions[].DNSName')
 ```
 
+#OpenStack
+
+```
+KUBERNETES_PUBLIC_ADDRESS=$(openstack server show controller0 -f shell |grep addresses | awk '{print $2}'| sed 's/"$//')
```

Comment:
this can be

```
KUBERNETES_PUBLIC_ADDRESS=$(openstack server show controller0 -f shell -c addresses | awk '{print $2}' | sed -e 's/"$//')
```

(the -c thing simplifies the output a bit more)